### ISSUE TITLE
Mini Map on Screen 

### ISSUE THREAD

**Issue by DarkxVoid-loop** [nipkownix/re4_tweaks#237]
### Describe the feature you'd like to see added

Put the map you have in the game
and put it as optional put it as a minimap at the top right of the screen

**Comment by Mister-Curious** [comment]
This would be possible but it would require editing all the EFF files for every room in the game. One limitation would be that the map would not be able to rotate with player movement.

**Comment by DarkxVoid-loop** [comment]
Mr Curioso so I think it is possible and he can move with the character as well with the quick turn, because with the compenion it was ok, just get the angle

**Comment by Mister-Curious** [comment]
yes the effect would stay on screen no matter what. I have created a similar effect that works this way, except that it is for an overlay, not a map).

**Comment by Mister-Curious** [comment]
Ok so I did a test run (see video below). There are some transparency issues when looking at the sky (maybe only because I have no sky in this room), but against all other models it seems to be ok. 

https://www.youtube.com/watch?v=JYhrnFgMsPg

While this seems like a novelty, I am not really sure that it is useful at all. It just seems like a big eyesore.

**Comment by DarkxVoid-loop** [comment]
I liked the result, I think you can fix the transparency, I would just add a round mini map DDS texture and zoom in on the room you are in

**Comment by Mister-Curious** [comment]
Thanks, but I am definitely not interested in how this looks. I don't even understand the purpose of it considering how small most of the areas are. Why would you want an overlay like that which is visible all the time when you can just press M for a much more detailed map with legends and interaction?

**Comment by DarkxVoid-loop** [comment]
If there was a way to do it in real time, it would be very good, as in RPG games where the map messes with your movement, I would like more in the mod part, that we change the maps and put fogs that you can't to find this minimap would be very useful
